<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name Server</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Name Data Base</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" rowspan="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="4" column="0" rowspan="2">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="5" column="1" rowspan="2" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>setWindowTitle</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
